testomania/src/main/java/com/earth/testomania/quiz_categories/AnimalsDestination.kt

```diff
@@ -31,8 +32,10 @@ class GetAnimalsQuizUseCse @Inject constructor(private val repository: OpenTdbRe
 @HiltViewModel
 class AnimalsQuizViewModel @Inject constructor(
     useCase: GetAnimalsQuizUseCse,
-    dispatcher: CoroutineDispatcher
+    dispatcher: CoroutineDispatcher,
+    unsplashRepo: UnsplashRepo,
 ) : DestinationViewModel(
     useCase,
-    dispatcher
+    dispatcher,
+    unsplashRepo
```

Comment:
Since `unsplashRepo` is the same for every destination (category) can't we inject it directly into `DestinationViewModel` instead of sending it as a parameter and duplicating it numerous times? 
I am not sure that we can, but maybe you know some solutions guys.
What do you think?
@kdiakonidze @shalva97 @Vazhapp